taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from .models import Manufacturer, Driver, Car
```

Comment:
Please add a blank line between different groups of imports and use full import instead of relative